.github/workflows/release-node-ui-dataviz.yml

```diff
@@ -0,0 +1,92 @@
+name: Release Node UI - Dataviz
+on:
+  workflow_call:
+      inputs:
```

Comment:
suggestion : project and build_folder can't be defaulted?

Reply:
removed inputs